cmd/gitopsUpdateDeployment.go

```diff
@@ -305,6 +335,30 @@ func cloneRepositoryAndChangeBranch(config *gitopsUpdateDeploymentOptions, gitUt
 	return nil
 }
 
+func downloadCACertbunde(customTlsCertificateLinks []string, gitUtils iGitopsUpdateDeploymentGitUtils, fileUtils gitopsUpdateDeploymentFileUtils) ([]byte, error) {
+	certs := []byte{}
```

Comment:
we could use `var certs []byte`, so that when there are no custom certificate links, slice will not be even initialized

Reply:
i always pass an empty byte list to the git client and then in the it git client identify if i should add it or not and hence always return an empty slice

Reply:
ok, makes sense. I just wanted to follow [this](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#declaring-empty-slices) **recommendation**